index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anthony Martin Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="header-logo">Anthony Martin</div>
+            <div class="header-menu-button">
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
```

Comment:
- To help improve the semantics of your website, I suggest that you wrap this section in a `nav` tag. Please fix this 🙏